### PR TITLE
use 1024 as total cpushare for a machine

### PR DESCRIPTION
**Pull request by vieux**
Signed-off-by: Victor Vieux <victorvieux@gmail.com>

### CODE REVIEW

File: cluster/swarm/node.go [hunk 1]
```diff
@@ -265,7 +265,7 @@ func (n *node) updateContainer(c dockerclient.Container, containers map[string]*
 		}
 		container.Info = *info
 		// real CpuShares -> nb of CPUs
-		container.Info.Config.CpuShares = container.Info.Config.CpuShares / 100.0 * n.Cpus
+		container.Info.Config.CpuShares = container.Info.Config.CpuShares / 1024.0 * n.Cpus
```
Comment (smothiki) [inline cluster/swarm/node.go]: container.Info.Config.CpuShares results in zero if ```container.Info.Config.CpuShares < 1024```

Comment (vieux) [inline cluster/swarm/node.go]: @smothiki right, I thought using 1024.0 was enough but as `container.Info.Config.CpuShares` is an int it doesn't work, I'll change thanks.

Comment (vieux) [inline cluster/swarm/node.go]: @smothiki it should be `container.Info.Config.CpuShares = container.Info.Config.CpuShares * 1024.0 / n.Cpus`

**Comment by vieux** [comment]
@smothiki @abronan @aluzzardi PTAL

**Comment by smothiki** [comment]
Looks good .But only issue is every data type ``int64`` So its very hard to get a fine grained CPU usage from the system . what happens if the system has 1 core ? or some one wants to run a batch process Job that should take very less CPU ? 

Also refresh time is 30 Seconds and what if a scheduler tries to launch a container in between .

**Comment by aluzzardi** [comment]
LGTM

**Comment by aluzzardi** [comment]
@smothiki That, unfortunately, is a limitation of the Docker API and there's nothing we can do about that.

**Pull request merged**